card.io/src/main/java/io/card/payment/CardScanner.java

```diff
@@ -45,6 +45,8 @@
     // value based on phone? or
     // change focus behavior?
 
+    private static final int DEFAULT_UNBLUR_DIGITS = 4;
```

Comment:
I did set this to -1 to disable blur now. Who knows how many numbers a creditcard has in the future :D